docs/quick-install.md

```diff
@@ -1,175 +1,184 @@
+### Step 2. Confirm your installation
 
-```sh
-& {Set-ExecutionPolicy Bypass -Scope Process -Force; $script = ((New-Object System.Net.WebClient).DownloadString('https://dl.signalfx.com/signalfx-agent.ps1')); $params = @{access_token = "YOUR_SIGNALFX_API_TOKEN"};; 
-ingest_url = "https://ingest.YOUR_SIGNALFX_REALM.signalfx.com"; api_url = "https://api.YOUR_SIGNALFX_REALM.signalfx.com"}; Invoke-Command -ScriptBlock ([scriptblock]::Create(". {$script} $(&{$args} @params)"))}
-```
+1. From the command line, enter the following command to confirm that your Smart Agent is functional:
 
-The agent will be installed as a Windows service and will log to the Windows Event Log.
+    ```sh
+    sudo signalfx-agent status
+    ```
 
+    The return should be similar to the following example:  
 
-### Step 2. Confirm your Installation 
+    ```sh
+    SignalFx Agent version:           4.7.6
+    Agent uptime:                     8m44s
+    Observers active:                 host
+    Active Monitors:                  16
+    Configured Monitors:              33
+    Discovered Endpoint Count:        6
+    Bad Monitor Config:               None
+    Global Dimensions:                {host: my-host-1}
+    Datapoints sent (last minute):    1614
+    Events Sent (last minute):        0
+    Trace Spans Sent (last minute):   0
+    ```
 
-To confirm the SignalFx Smart Agent installation is functional on either platform, enter:
+2. Enter the following commands to verify the installation:
 
-```sh
-sudo signalfx-agent status
-```
+    | Command | Description   |
+    |---|---|
+    | <code>signalfx-agent status config</code>   | This command shows resolved config in use by the Smart Agent. |
+    | <code>signalfx-agent status endpoints</code>  | This command shows discovered endpoints.  |
+    | <code>signalfx-agent status monitors</code>  | This command shows active monitors.  |
+    | <code>signalfx-agent status all</code>  | This command shows all of the above statuses. |
 
-The response you will see is similar to the one below:
+***
 
-```sh
-SignalFx Agent version:           4.7.6
-Agent uptime:                     8m44s
-Observers active:                 host
-Active Monitors:                  16
-Configured Monitors:              33
-Discovered Endpoint Count:        6
-Bad Monitor Config:               None
-Global Dimensions:                {host: my-host-1}
-Datapoints sent (last minute):    1614
-Events Sent (last minute):        0
-Trace Spans Sent (last minute):   0
-```
+### Troubleshoot the Smart Agent installation
 
-To verify the installation, you can run the following commands:
+If you are unable to install the Smart Agent, consider the following issues:
 
-```sh
-signalfx-agent status config - show resolved config in use by agent
-signalfx-agent status endpoints - show discovered endpoints
-signalfx-agent status monitors - show active monitors
-signalfx-agent status all - show everything
-```
+***
+ 
+#### Update your realm
```

Comment:
If the install from the UI, they should never have to do anything with realms, and even if they did, these instructions don't really tell them what to fix or how to fix it. Ditto for those endpoints. 

In fact, this is why we are now people to install from the UI -- so they don't have to deal with any of this stuff.

Reply:
The doc site currently says, "By default, the Smart Agent will send data to the us0 realm. If you are not in this realm, you will need to explicitly set the signalFxRealm option in your config like this:..." 

This statement is confusing for a number of reasons, including your previous comment about installing from the UI where the realm is already populated. Does the above statement imply that if the pre-populated command in the UI says us0, then you need to confirm elsewhere that is indeed your realm, and not simply a default / placeholder realm? Similarly, to me, the statement implies that you can possibly run the install command without specifying a realm because, by default, the installation will be configured to the us0 realm.

Long story short, I think maybe it's best to remove this section entirely. What do you think?

Reply:
Yes, everything related to realm should go. This was here because we provided template for them to enter at the command line themselves. Now we don't so this should not matter to them.

Reply:
The "update your realm" section has been removed.